packages/components/organisms/f-offers/src/utils/helpers.js

```diff
@@ -0,0 +1,24 @@
+
+const setCookie = (name, value, minutes = 1) => {
+    let expires = '';
+    const millisecondsPerMinute = 60 * 1000;
+    if (minutes) {
+        const date = new Date();
+        date.setTime(date.getTime() + (minutes * millisecondsPerMinute));
+        expires = `; expires=${date.toUTCString()}`;
+    }
+    document.cookie = `${name}=${(value || '') + expires}; path=/`;
+};
+
+export const pushToDataLayer = (category, action, label) => {
+    window.trak.event({
+        event: 'trackEvent',
+        category,
+        action,
+        label
+    });
+};
+
+export const setGtmEventCookie = (category, action, label) => {
+    setCookie('je-gtm-event', `${category}|${action}|${label}`);
```

Comment:
I'd look at using `js-cookie` here. We're already using it as a peer dependency in `f-cookie-banner`, `f-braze-adapter` and `f-consumer-oidc`.

Reply:
thanks for the tip

Reply:
updated to now use `js-cookie`